### ISSUE TITLE
Please implement brackets.io like preview

### ISSUE THREAD

**Issue by sindzicat** [ritwickdey/vscode-live-server#190]
Hello.

I like html preview, realized in brackes (brackets.io). When I click on some tag in html file, this block is highlighted in preview. Preview runs in Chrome and (experimental) other browers. 

This preivew isn't realized in vscode yet ([issue](https://github.com/Microsoft/vscode/issues/12080)). I think it will be cool if such preivew will be realized in your extension.

--Andrey

**Comment by dalDevelo** [comment]
Agreed - Would like to see this implementation. Brackets has got it right.

**Comment by muntajibuddin** [comment]
True. This is a great feature in Bracket that VSCode lacks.

**Comment by qbodk** [comment]
The feature is a necessity. 
every people who has used bracket knows it.
